vms/administrator/tests/test_formFields.py

```diff
@@ -87,12 +105,17 @@ def check_job_form_values(self, job):
         :param job: Iterable consisting values for job.
         """
         settings = self.settings
+        self.driver.implicitly_wait(8)
```

Comment:
Put implicit wait time as a variable(instead of hardcoded time value).

Reply:
Shall I make a single global time variable to do the work for all the implicit waits?

Reply:
Yes